### DISCUSSION TITLE
How to help?

### DISCUSSION THREAD

**Discussion by Nichtraucher** [nextcloud/cookbook#2060] [General]
Hey!

How do you deal with websites that are not being processed properly - Is there a dedicated topic or issue where I can report them?

cheers

**Comment by christianlupus** [comment]
You can report pages that are not working but we cannot guarantee them to work. If the pages are rejecting our requests actively or not providing schema.org meta data, we are currently out of luck.

What pages are you referring to?

**Comment by Nichtraucher** [comment]
Many thanks for your support.

I gather most recipes from wdr.de such as this one: https://www1.wdr.de/verbraucher/rezepte/alle-rezepte/geriffelte-wurzelpommes-mit-sour-cream-dipp-100.html I noticed the Schema references in the source code:

`"@context" : "https://schema.org/",
"@type" : "Article",
"image" : [ {
"@type" : "ImageObject",
"contentUrl" : "https://www1.wdr.de/mediathek/video/sendungen/der-vorkoster/viel-fuer-wenig-geriffelte-wurzelpommes-100~_v-gseagaleriexl.jpg",
"height" : 900,
"width" : 1600,
"author" : "WDR",
"name" : "Geriffelte Wurzelpommes mit Sour-Cream-Dipp",
"datePublished" : "2023-09-08",
"description" : "Das Bild zeigt das fertige Gericht Geriffelte Wurzelpommes mit Sour-Cream-Dipp."
.
.
.
`

I'm getting "no parser could be found" with all recipes. I'm using Nextcloud 27.1.5 and cookbook 0.10.5.

If necessary, I'll try to contact wdr.de and inform them about the issue.

**Reply by christianlupus** [reply]
Unfortunately, I am abroad currently on a business trip. With the mobile office I can only barely debug this. I guess the problem is a known shortcomings that should be addressed soon. But let me look at this in detail before.

**Reply by seyfeb** [reply]
The JSON object you are referring to is not a `Recipe` object but has type `Article`. You will find that there are, e.g., no recipe instructions, nutrition, etc. listed. The cookbook app currently supports importing `schema.org/Recipe` objects from ld+json. More "intelligent" or complex parsers could be implemented but are currently missing.

**Reply by christianlupus** [reply]
You seem to have looked at the sources. I just looked at the quoted song that was ending in an ellipsis. But then, no, unfortunately, no partying is possible ATM.

**Comment by Nichtraucher** [comment]
Thanks for the informative reply! I'll email the WDR anyway.  :-)


> I just looked at the quoted song that was ending in an ellipsis. 

Quoted song? Ellipsis?

**Reply by christianlupus** [reply]
Oh, man. This is my trying to make fun of me. Sorry for that. 

I meant I just looked at the quoted code you provided. I did not look at the raw source. Further your quoted code was abbreviated and had an ellipsis (...) at the end. This I was not sure, if there would maybe be something hidden there that could be used.
